addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -555,13 +556,18 @@ Object.assign(MessageList, {
         haveMessagesMarkAsReadIcon: false,
         haveMessagesReplyIcon: false,
         order: 'asc',
+        scrollableElement: () => {},
```

Comment:
Default should return `this.el`. I'm not sure if you'll be able to properly bind it here though, I guess maybe we need a private function that does it.

Reply:
@seb-odoo . here we are unable to get `this.el`, as `this` is undefined because it is just a prop defination not a declaration. and some confusion in this private function statement, if we can have `this` then only we can make call the private function. 
I think its a same case.